lib/rouge/lexers/llvm.rb

```diff
@@ -14,20 +14,7 @@ class LLVM < RegexLexer
       string = /"[^"]*?"/
       identifier = /([-a-zA-Z$._][-a-zA-Z$._0-9]*|#{string})/
 
-      def self.keywords
-        Kernel::load File.join(Lexers::BASE_DIR, "llvm/keywords.rb")
-        keywords
-      end
-
-      def self.instructions
-        Kernel::load File.join(Lexers::BASE_DIR, "llvm/keywords.rb")
-        instructions
-      end
-
-      def self.types
-        Kernel::load File.join(Lexers::BASE_DIR, "llvm/keywords.rb")
-        types
-      end
+      require_relative "llvm/keywords"
```

Comment:
Similar to before, we'd like to avoid loading these large keywords files for users who aren't highlighting LLVM. Though now I look at this it appears to have been a copy paste job and it will absolutely load that file multiple times 🤦

Reply:
The old code is wrong. It shouldn't use `load`. If that alone gets fixed, great... but seriously, The strategies you used 10 years ago to speed things up aren't the strategies you need today. Case in point:

```ruby
$-w = nil

$: << "lib"
require "rouge"
keywords = Dir["lib/rouge/lexers/*/keywords.rb"]

iters = 30

t0 = Time.now
iters.times do
  keywords.each { |f| Kernel.load f }
end
t1 = Time.now

puts "Average time to load all keywords.rb: %8.5fs" % [(t1 - t0) / iters]

# Average time to load all keywords.rb:  0.00812s
```

It costs nearly nothing to load those keyword files... granted, every machine is different, but still. I suggest you measure and reevaluate whether self modifying code is worth it. I had thousands of lines of warnings when I was first evaluating whether to switch to rouge or not. Literally thousands. Those warnings have a cost too.

Reply:
If you really want to make rouge faster, don't load everything. Load (roughly) nothing. Push it towards a lazy loading system. But if you're going to load everything, let ruby do it and let it do it properly (and only once)